docs/examples/301/modules-vwan-to-vnet-s2s-with-fw/main.bicep

```diff
@@ -0,0 +1,206 @@
+targetScope = 'subscription'
+
+param location string {
+  default: 'westeurope'
+  metadata: {
+    description: 'Specify the location for the hub Virtual Network and its related resources'
+  }
+}
+param vwanlocation string {
+  default: 'eastus'
+  metadata: {
+    description: 'Specify the location for the vWAN and its related resources'
+  }
+}
+param nameprefix string {
+  default: 'contoso'
+  metadata: {
+    description: 'Specify the name prefix for all resources and resource groups'
+  }
+}
+param psk string {
+  secure: true
+  metadata: {
+    'description': 'PSK to use for the site to site tunnel between Virtual Hub and On-Prem VNet'

```

Comment:
What does PSK stand for? May be good to spell it out in the description

Reply:
pre-shared key, I have updated the description